src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2931,4 +2931,12 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="UpdateAvailable.Title" xml:space="preserve">
     <value>Update available</value>
   </data>
+  <data name="FileExplorerPreview_Toggle_Monaco_Max_File_Size.Header" xml:space="preserve">
+    <value>Maximum file size to preview</value>
+    <comment>Size refers to the disk space used by a file</comment>
+  </data>
+  <data name="FileExplorerPreview_Toggle_Monaco_Max_File_Size.Description" xml:space="preserve">
+    <value>The maximum size a file can have to get displayed in bytes. This is a safety mechanism to prevent loading big files into RAM.</value>
```

Comment:
```suggestion
    <value>The maximum size, in bytes, for files to be displayed. This is a safety mechanism to prevent loading large files into RAM.</value>
```
The common word seems to be "large"